script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -140,13 +140,13 @@ I mean, even with a gag order, there's going to be leaks, right?"
 "So long as we have a cover story, the rest is just a matter of legwork.
  Why d'ya think my old man's been leaning on people left and right?
  All you gotta do is put pressure on the guys at the top.
- Doesn't make a fat bit of difference if the small fry or middle managers get suspicious.
+ Doesn't make a fat bit of difference if the small fries or middle managers get suspicious.
```

Comment:
we talked about this during ross's PR but yusuke and ross prefer "small fry" and it seems to be a valid plural form so revert.